roles/db/tasks/main.yml

```diff
@@ -63,6 +63,16 @@
     type: database
     role: "{{ db_user }}"
 
+- name: Grant schema public to the user.
+  become: true
+  become_user: postgres
+  community.postgresql.postgresql_privs:
+    db: "{{ db_name }}"
+    privs: ALL
```

Comment:
@autn what error does this step solve? Is there a related stackoverflow answer or GitHub you could link me to?

What I'm wondering in particular is whether we should provide `ALL` privileges or whether a more limited set of privileges is appropriate

Reply:
One thing I'm wondering here is whether we should use `USAGE` instead of `ALL`...I'm not sure and I can't find documentation explaining the difference. Would `USAGE` work for the repo?

Reply:
@YPCrumble I'll try to re-build a server to make sure it will work or not!